testing-modules/junit/src/main/java/com/baeldung/Main.java

```diff
@@ -0,0 +1,7 @@
+package com.baeldung;
+
+public class Main {
+	public static void main(String[] args) {
```

Comment:
Indentation in Java  code should always use spaces (four per level) and not tabs.